sunpy/map/compositemap.py

```diff
@@ -64,8 +64,6 @@ class CompositeMap(object):
     Examples
     --------
     >>> import sunpy.map
-    >>> import sunpy.data
-    >>> sunpy.data.download_sample_data(overwrite=False)   # doctest: +SKIP
     >>> import sunpy.data.sample
```

Comment:
I don't think you need to skip any of these anymore because it will already find the files locally and will not download them.